mirar/pipelines/sedmv2/load_sedmv2_image.py

```diff
@@ -197,10 +239,10 @@ def load_raw_sedmv2_mef(
             split_data = split_data[1:]
             split_headers = split_headers[1:]
         header, split_headers = clean_science_header(header, split_headers, is_mode0)
-    elif check_header["IMGTYPE"] in ["flat", "bias"]:
-        header, split_headers = clean_cal_header(header, split_headers[0], path)
+    elif check_header["IMGTYPE"] in ["flat", "bias", "dark", "unknown", "illum"]:
+        header, split_headers = clean_cal_header(header, split_headers, path, is_mode0)
     else:
-        logger.debug("Unexpected IMGTYPE. Is this a dark?")
+        logger.debug("Unexpected IMGTYPE.")
```

Comment:
Maybe should be `logger.debug("Unexpected IMGTYPE: {check_header['IMGTYPE']})` ?